website/docs/addon_applications_admin.md

```diff
@@ -1,26 +1,230 @@
+![settings_applications](assets/applications/settings_addapplication.gif)
+
+1. **Add version** in case you want this version to be selectable individually. This is typically used for bigger releases that might not be fully compatible with previous versions. Keep in mind that if you add the latest version of an Application that is not yet part of the official AYON release, you might run into problems with integration. We test all the new software versions for compatibility and most often, smaller or bigger updates to AYON code are necessary to keep everything running.
+![settings_applications](assets/applications/settings_addappversion.gif)
+
+</details>
+
+### What is host name used for ?
+
+<details><summary>What is host name used for ?</summary>
+
+Applications addon uses host name to match applications to their host integration addons.
+different applications can have the same host name as long as they are related to the same addon.
+And, host name is defined in the addon's client code. e.g. [nuke's addon host name](https://github.com/ynput/ayon-nuke/blob/7a08f570512d5bd92caf014ba159889cfd912d38/client/ayon_nuke/addon.py#L13).
+
+![](assets/applications/different_apps_same_host.png)
+
+:::tip
+Host name of the additional application can add application we don't have integrated yet (e.g. GIMP) and add gimp addon integration.
+So, you'd add gimp as additional app and AYON will be able to match it with your addon with gimp host name.
+:::
+
+Also note that, Host name is used in different places in AYON settings for profile filtering. 
+e.g. `Hosts` list in [Tools Definitions](addon_applications_admin#tools-definitions) settings expects host names.
+
+</details>
+
+### What does AYON offer to customize the application environments ?
+
+<details><summary>What does AYON offer to customize the application environments ?</summary>
+
+> Originally answered by Roy Nieterau on ynput's forums [Maya - User prefs and studio prefs](https://community.ynput.io/t/maya-user-prefs-and-studio-prefs/1683/3)
+
+1. Global Environment Variables: `ayon+settings://core/environments`
+2. Per application group environment variables. e.g. maya: `ayon+settings://applications/applications/maya/environment`
+3. Per application variant environment variables, e.g. maya 2025: `ayon+settings://applications/applications/maya/variants/0/environment`
+4. Tools environment variables.
+
+Tools are special in that you can set tools to apply to only to certain contexts, e.g. only certain assets or shots instead of project-wide.
+
+</details>
+
+### What is the syntax AYON uses for defining environments in settings ?
```

Comment:
I wonder if it _may_ help linking that this uses [`acre` to construct the environment variables from the dicts](https://github.com/ynput/acre).

Because that package may then explain that we support [Platform specific assignments](https://github.com/ynput/acre?tab=readme-ov-file#platform-specific-paths-acreparse).

Also, it may be worth mentioning that we **recommend** the list assignment like you're doing in the TIP here because setting e.g. `"MY_VARIABLE": "A;B;C"` would work on Windows, but not on UNIX because there it the path separator is `:` instead of `;` so should be `A:B:C` on UNIX. Using the "MY_VARIABLE": ["A", "B", "C"]` automatically resolves it to the platform-specific separator.